### ISSUE TITLE
PRODFILES_slave Instance

### ISSUE THREAD

**Issue by leobardoramon** [gosha1128/NYUAMS#566]
Andrew is wondering if it would be a good idea to have a slave instance of PRODFILES that is always 24 hours behind...this way if we need to take a snapshot of PRODFILES for dev purposes, we can refer to the slave instance. It seems like a helpful failsafe anyway.

Thoughts?

**Comment by gosha1128** [comment]
Yes.  Just plan for the cost of doing this so you aren't surprised :)

Also, remember that whenever you use a data backup - you should backup both
PRODFILES and RDS at the same time.  This means obviously, when you use a
backup, you use a backup pair.

Does that make sense ?

George

**Comment by leobardoramon** [comment]
I'm wondering if it would be just as effective to schedule "PRODFILES" snapshots nightly, back-to-back with rds snapshots...
